server/routes/index.js

```diff
@@ -2,13 +2,20 @@
 const router = require('express').Router();
 
 // modular routers
-const therapistRoute = require('./therapistRoute')
-const pathTwo = require('./pathTwo')
+const therapist = require('./therapist')
+const patient = require('./patient')
+const routetemplate = require('./routetemplate') 
 
 // routing here
 router
-  .use('/therapist', therapistRoute)
-  .use('/pathTwo', pathTwo)
+  .use('/therapist', therapist)
+  .use('/patient', patient)
+  .use('/routetemplate', routetemplate)
+
+// Send along any errors
+router.use((err, req, res, next) => {
```

Comment:
We should not need this, there is a universal error handler in `app.js` already?